crates/modelardb_server/src/context.rs

```diff
@@ -445,19 +446,15 @@ mod tests {
             .unwrap();
 
         // Both a normal table and a model table should be created.
-        assert!(
-            context
-                .check_if_table_exists(test::NORMAL_TABLE_NAME)
-                .await
-                .is_err()
-        );
+        assert!(context
```

Comment:
It seems that a lot of the changes revert the formatting that was done recently with Rustfmt when we updated the version of Rust. Maybe reformat again to ensure that the formatting is correct this time so we do not keep reformatting the same code every PR.

Reply:
Fixed and updated editor configuration to use edition 2024 for formatting.